app/models/message.rb

```diff
@@ -1,4 +1,5 @@
 class Message < ApplicationRecord
   belongs_to :user
   validates :body, presence: true, length: { minimum: 1, maximum: 1000 }
+  scope :recent, -> { order(:created_at).last(12) }
```

Comment:
Do you want to return the order as desc as well?

Reply:
I don't think that's necessary as the messages currently render from top to bottom with the bottom being the 'largest' order.